e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/SettingsPageAO.java

```diff
@@ -1175,6 +1175,30 @@ public PreferencesAO setPreference(String preference, String value, boolean eyeI
             return this;
         }
 
+        public String getPreference(String preference) {
+            searchPreference(preference);
+            String[] strings = context().$(byClassName("CodeMirror-code"))
+                    .findAll(byClassName("CodeMirror-line")).texts().toArray(new String[0]);
```

Comment:
Why should we result in String[] instead of List<String>?

Reply:
Done